packages/visx-stats/src/util/computeStats.ts

```diff
@@ -3,8 +3,23 @@ import { BoxPlot, BinDatum } from '../types';
 export default function computeStats(numericalArray: number[]) {
   const points = [...numericalArray].sort((a, b) => a - b);
   const sampleSize = points.length;
-  const firstQuartile = points[Math.round(sampleSize / 4)];
-  const thirdQuartile = points[Math.round((3 * sampleSize) / 4)];
+
+  function calcMedian(dataSet: number[]) {
```

Comment:
since it has no close dependencies, this could be pulled outside the `computeStats` function